nano/rpc_test/rpc.cpp

```diff
@@ -977,6 +977,7 @@ TEST (rpc, wallet_create_seed)
 	nano::system system (24000, 1);
 	scoped_io_thread_name_change scoped_thread_name_io;
 	nano::raw_key seed;
+	seed.data = 1;
```

Comment:
May be better use nano::keypair that generates ramdom value?

Reply:
Good point, calling `nano::random_pool::generate_block` to generate random data now.